src/script/conversation/ConversationVerificationStateHandler/Proteus/ProteusStateHandler.ts

```diff
@@ -182,10 +183,8 @@ export class ProteusConversationVerificationStateHandler {
   ): boolean {
     // We want to process only Proteus and MLS conversations
     if (isProteusConversation(conversationEntity) || isMixedConversation(conversationEntity)) {
```

Comment:
**question**: Not relevant to changes, but the comment says we process only proteus and mls, but the check lets proteus and mixed conversations in, is that intentional? As a recap, mixed conversation is really a proteus conversation that is being migrated, it has a MLS group under the hood where proteus conversation participants are being added during migration (It will change the protocol to MLS once the migration is finalised)